src/molecules/Tabs.tsx

```diff
@@ -77,29 +137,36 @@ const Tabs = withRouter(({ match }) => {
               I'm description Lorem ipsum dolor sit amet consectetur adipisicing
               elit. Quo sapiente tenetur cum, quis, quia veniam quod fugit
               eligendi cupiditate vero, sequi asperiores cumque eos ipsum
-              pariatur alias dolore saepe unde.
+              pariatur alias dolore saepe unde. I'm description Lorem ipsum
+              dolor sit amet consectetur adipisicing elit. Quo sapiente tenetur
+              cum, quis, quia veniam quod fugit eligendi cupiditate vero, sequi
+              asperiores cumque eos ipsum pariatur alias dolore saepe unde.
             </StyledP>
           )}
         />
         <Route
           path={`${match.url}/info`}
           render={() => (
             <StyledP>
-              I'm info Lorem ipsum, dolor sit amet consectetur adipisicing elit.
-              Inventore dolorum similique officia ipsa magni, expedita
-              consequuntur pariatur perferendis ipsam at reprehenderit,
-              voluptate id, debitis amet facilis voluptatibus qui illo commodi.{' '}
+              <StyledTable>
```

Comment:
The same comment as related to reviews: I think this should be a separate component next to the product page component. This would make components much more manageable.